ui/raidboss/data/07-dt/dungeon/origenics.ts

```diff
@@ -359,9 +372,11 @@ const triggerSet: TriggerSet<Data> = {
       outputStrings: {
         right: {
           en: '(Far North lane: Knockback Right ==>)',
+          fr: '(Ligne nord éloignée : Poussée droite ==>)',
```

Comment:
```suggestion
          en: '(Far North lane: Knockback Right) ==>',
          fr: '(Ligne nord éloignée : Poussée droite) ==>',
```

Given the recent discussion/changes in #465, we should move the arrows outside of the parens so they begin/end the string to avoid tts jank.  If you wouldn't mind fixing the `en` version with this PR too, much appreciated. 😄   [Note to self/whomever: we might want to add a unit test for this.]